script/model/model_server.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python3
+"""
+This file contains the python model server implementation.
+
+The server should be stateless but with caching of models.
+The message format that the model-server expects should be kept consistent with Messenger class in
```

Comment:
nit: can we consistently find and replace

model-server
modelserver
model server

etc

to be a single spelling? Maybe ModelServer?